lib/GithubHandler.js

```diff
@@ -177,15 +177,15 @@ GithubHandler.prototype.pullRequestHandler = function(event, cb) {
     type: 'pull_request', // also in event.event
     service: 'github',
     host: undefined,
+    branch: event.payload.pull_request.head.ref,
```

Comment:
The merged refspec is based on the PR number. For example: `refs/pull/18/merge` for PR #18. So we don't necessarily need to record it.

`pull_request.head.ref` is the name of the branch, not the sha ref of the commit. That's in `pull_request.head.sha`